packages/@aws-cdk/aws-lambda-nodejs/README.md

```diff
@@ -132,7 +132,8 @@ Docker container even if `esbuild` is available in your environment. This can be
 ## Configuring `esbuild`
 
 The `NodejsFunction` construct exposes some [esbuild options](https://esbuild.github.io/api/#build-api)
-via properties under `bundling`: `minify`, `sourceMap`, `target` and `loader`.
+via properties under `bundling`: `minify`, `sourceMap`, `target`, `loader`, `logLevel` and
+`keepNames`.
```

Comment:
Maybe it's not necessary to list all of these here.

Reply:
done, can you reapprove? thx.